site/src/components/Dialogs/DeleteDialog/DeleteDialog.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ReactNode } from "react"
+import { ConfirmDialog } from "../ConfirmDialog/ConfirmDialog"
+
+export interface DeleteDialogProps {
+  isOpen: boolean
+  handleConfirm: () => void
```

Comment:
Why not keep using the `onSomeAction` pattern? I think this is the most used one from libs.

Reply:
The `DeleteWorkspaceDialog` component used `handle`, but I'd like it to be consistent so I'll switch it